google/cloud/security/inventory/inventory_loader.py

```diff
@@ -191,98 +191,56 @@ def _build_pipelines(cycle_timestamp, configs, **kwargs):
 
-    # The order here matters, e.g. groups_pipeline must come before
-    # group_members_pipeline.
-    pipelines = [
-        load_orgs_pipeline.LoadOrgsPipeline(
-            cycle_timestamp,
-            configs,
-            crm_v1_api_client,
-            kwargs.get(organization_dao_name)
-        ),
-        load_org_iam_policies_pipeline.LoadOrgIamPoliciesPipeline(
-            cycle_timestamp,
-            configs,
-            crm_v1_api_client,
-            kwargs.get(organization_dao_name)
-        ),
-        load_projects_pipeline.LoadProjectsPipeline(
-            cycle_timestamp,
-            configs,
-            crm_v1_api_client,
-            kwargs.get(project_dao_name)
-        ),
-        load_projects_iam_policies_pipeline.LoadProjectsIamPoliciesPipeline(
-            cycle_timestamp,
-            configs,
-            crm_v1_api_client,
-            kwargs.get(project_dao_name)
-        ),
-        load_projects_buckets_pipeline.LoadProjectsBucketsPipeline(
-            cycle_timestamp,
-            configs,
-            gcs_api_client,
-            kwargs.get(project_dao_name)
-        ),
-        load_projects_buckets_acls_pipeline.LoadProjectsBucketsAclsPipeline(
-            cycle_timestamp,
-            configs,
-            gcs_api_client,
-            kwargs.get('bucket_dao')
-        ),
-        load_projects_cloudsql_pipeline.LoadProjectsCloudsqlPipeline(
-            cycle_timestamp,
-            configs,
-            cloudsql.CloudsqlClient(),
-            kwargs.get('cloudsql_dao')
-        ),
-        load_forwarding_rules_pipeline.LoadForwardingRulesPipeline(
-            cycle_timestamp,
-            configs,
-            compute.ComputeClient(),
-            kwargs.get('fwd_rules_dao')
-        ),
-        load_folders_pipeline.LoadFoldersPipeline(
-            cycle_timestamp,
-            configs,
-            crm.CloudResourceManagerClient(version='v2beta1'),
-            dao
-        ),
-        load_bigquery_datasets_pipeline.LoadBigQueryDatasetsPipeline(
-            cycle_timestamp,
-            configs,
-            bq.BigQueryClient(),
-            dao
-        ),
-        load_firewall_rules_pipeline.LoadFirewallRulesPipeline(
-            cycle_timestamp,
-            configs,
-            compute.ComputeClient(version='beta'),
-            kwargs.get(project_dao_name)
-        ),
-    ]
-
-    if configs.get('inventory_groups'):
-        if util.can_inventory_groups(configs):
-            admin_api_client = ad.AdminDirectoryClient()
-            pipelines.extend([
-                load_groups_pipeline.LoadGroupsPipeline(
-                    cycle_timestamp,
-                    configs,
-                    admin_api_client,
-                    dao),
-                load_group_members_pipeline.LoadGroupMembersPipeline(
+
+    inventory_pipeline_configs = file_loader.read_and_parse_file(
+        'google/cloud/security/inventory/inventory_pipeline_configs.yaml')
```

Comment:
Just a note, though you may already be aware -- you might need to indicate the yaml as a data file in setup.py, if forseti_inventory can't find the yaml.